pkg/tables/tables.go

```diff
@@ -31,11 +31,17 @@ var (
 	}
 	VolumeConverter = MustConverter(Volume)
 
-	Image = [][]string{
+	ImageAcorn = [][]string{
 		{"Repository", "{{if eq .Repository \"\"}}<none>{{else}}{{.Repository}}{{end}}"},
 		{"Tag", "{{if eq .Tag \"\"}}<none>{{else}}{{.Tag}}{{end}}"},
 		{"Image-ID", "{{trunc .Name}}"},
 	}
+	ImageAcornConverter = MustConverter(ImageAcorn)
```

Comment:
this converter is not used. please delete.

Reply:
removed